app/services/real_cycle_schedule.rb

```diff
@@ -0,0 +1,18 @@
+class RealCycleSchedule
+  attr_reader :year
+
+  def initialize(year)
+    @year = year
+  end
+
+  def cycle_dates
```

Comment:
Rather than return a hash should we have methods for each event e.g.

```
def find_reopens
   Date.new(year - 1, 10, 6)
end
```

The issue here would be this would be a breaking change - is this beyond the scope of the exercise or worth doing regardless?

Reply:
I think a hash of dates is more readable? Maybe no need in this PR. If we wanted to move to a more method-y interface we could swap in a struct — they support `[:property]` and `.property` addressing